extensions/rocksdb-repos/FlowFileRepository.h

```diff
@@ -122,6 +126,21 @@ class FlowFileRepository : public core::Repository, public std::enable_shared_fr
     else
       return false;
   }
+
+  virtual bool MultiPut(const std::vector<std::tuple<std::string, const uint8_t *, size_t>> data) {
```

Comment:
this looks like good functionality. Why is this WIP? have you proven it in terms of performance improvements?

Reply:
Yes, it's way faster, will share some results. 

WIP as I'm not yet happy with the API, plan some improvements before considered as final.